app-crypt/veracrypt/veracrypt-1.19-r1.ebuild

```diff
@@ -45,6 +45,15 @@ PATCHES=(
 	"${FILESDIR}/${P}-remove-packaging-from-makefile.patch"
 )
 
+src_prepare() {
+	default
+	WX_BASENAME=$( ${EROOT}usr/bin/wx-config --basename )
+	if [[ ${WX_BASENAME} == "wx_baseu" ]] ; then
```

Comment:
Conditional patching is strongly discouraged. Can't you get the patch to be correct unconditionally? Then you should submit it upstream.